src/aro/Preprocessor.zig

```diff
@@ -1703,7 +1703,10 @@ fn expandFuncMacro(
                     else
                         &[1]TokenWithExpansionLocs{tokFromRaw(raw_next)},
                     .macro_param, .macro_param_no_expand => getPasteArgs(args.items[raw_next.end]),
-                    .keyword_va_args => variable_arguments.items,
+                    .keyword_va_args => if (variable_arguments.items.len == 0) blk: {
+                        try pp.err(raw_next, .no_argument_variadic_macro, .{});
+                        break :blk &[1]TokenWithExpansionLocs{.{ .id = .placemarker, .loc = .{ .id = .generated } }};
```

Comment:
Is this pointer valid when using it with a `break` statement like this? I'm not totally sure what the lifetime of a stack object like this is if it's inside an if statement.

Reply:
This should be fine because it is a comptime known value but generally I don't think this would be valid.